src/main/java/com/adyen/service/Payment.java

```diff
@@ -48,7 +51,7 @@ public Payment(Client client) {
      * @return
      * @throws Exception
```

Comment:
Could you please update the javadoc here?
It is also fine if you just remove "return" and "throws".

Reply:
Ok! I will remove all ("return" and "throws").